qsirecon/tests/data/mrtrix_multishell_msmt_hsvs_outputs.txt

```diff
@@ -36,9 +36,10 @@ logs/CITATION.md
 logs/CITATION.tex
 sub-01
 sub-01/anat
-sub-01/anat/sub-01_space-ACPC_seg-hsvs_probseg.nii.gz
+sub-01/anat/sub-01_space-T1w_seg-hsvs_probseg.nii.gz
```

Comment:
This now gets written out at `sub-01/anat/sub-01_seg-hsvs_probseg.nii.gz` (no space entity) because of the source file. @mattcieslak what would you prefer? Should I hardcode in `space-ACPC` like we had before or leave it without a space entity?

Reply:
Okay, I'll make that change.